packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -219,7 +218,7 @@ const DataGrid = injectIntl((props) => {
   // cells. In order to do so, these datagrid methods will be exposed to
   // allow those components to request focus change.
   useImperativeHandle(
```

Comment:
Do we still need the imperative handle?  With a reference to the DataGrid component, couldn't you just called the methods directly without needing to forward refs?